src/python/foglamp/configuration_manager.py

```diff
@@ -0,0 +1,330 @@
+# -*- coding: utf-8 -*-
+"""FOGLAMP_PRELUDE_BEGIN
+{{FOGLAMP_LICENSE_DESCRIPTION}}
+
+See: http://foglamp.readthedocs.io/
+
+Copyright (c) 2017 OSIsoft, LLC
+License: Apache 2.0
+
+FOGLAMP_PRELUDE_END
+"""
+
+import logging
+import psycopg2
+import aiopg.sa
+import sqlalchemy as sa
+import asyncio
```

Comment:
move to main itself, so that when plugged you just remove that block and you don't need to revisit this..